openslides/utils/autoupdate.py

```diff
@@ -97,9 +97,15 @@ def send_object(cls, object_url):
         object_url.
         """
         # Join network location with object URL.
+        if RUNNING_HOST == '0.0.0.0':
+            # Windows can not connect to 0.0.0.0. So connect to localhost instead
+            local_host = 'localhost'
```

Comment:
you can do it if you like. And it is not localhost but the host where the local application runs. If the application runs on 192.168.0.1, then local_host is the local host.